frontend/src/modules/_shared/Figure.tsx

```diff
@@ -190,24 +192,6 @@ export class Figure {
         merge(this._plotLayout, patch);
     }
 
-    makePlot(plotArgs?: Omit<PlotParams, "data" | "layout">): React.ReactNode {
-        const config = plotArgs?.config ?? {
-            displaylogo: false,
-            responsive: true,
-            modeBarButtonsToRemove: ["toImage", "sendDataToCloud", "resetScale2d"],
-        };
```

Comment:
You moved the default config to your plot component, so it's not necessary to choose either strategy, but you should also here adhere to one: `makePlotProps` vs. `makeLayout/makeData`.